core/orchestrator.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/livepeer/lpms/stream"
 )
 
-var MaxTranscodeSessions = 10
+var MaxSessions = 0
```

Comment:
Should this be in `livepeernode.go` since it's shared between B/O now?

Reply:
moved